packages/datadog-plugin-rhea/src/consumer.js

```diff
@@ -0,0 +1,56 @@
+'use strict'
+
+const ConsumerPlugin = require('../../dd-trace/src/plugins/consumer')
+const { storage } = require('../../datadog-core')
+
+class RheaConsumerPlugin extends ConsumerPlugin {
+  static get name () { return 'rhea' }
+
+  constructor (...args) {
+    super(...args)
+
+    // TODO: Remove this as it has no use.
```

Comment:
Should this be removed now?

Reply:
I guess this could have been better worded, but it's a breaking change so it can only be removed in a major. Will update to reflect that.

Reply:
Actually I'm just gonna remove the TODO as I already put in the 4.0 plan to review AMQP integrations.